csm_web/scheduler/admin.py

```diff
@@ -81,11 +81,11 @@ def has_delete_permission(self, request, obj=None):
 
 @admin.register(Student)
 class StudentAdmin(CoordAdmin):
```

Comment:
Not important for this PR, but in the future I'd like to pull permissioning out into a separate mixin class instead of having everything inherit from CoordAdmin, which is somewhat semantically confusing